lib/rails/templates/alchemy.rb

```diff
@@ -1,6 +1,6 @@
 # This rails template installs Alchemy and all depending gems.
 require File.expand_path('../../../alchemy/version', __FILE__)
 
-gem 'alchemy_cms', github: 'magiclabs/alchemy_cms', branch: 'master'
-gem 'alchemy-devise', github: 'magiclabs/alchemy-devise', branch: 'master'
+gem 'alchemy_cms', github: 'magiclabs/alchemy_cms', branch: '3.0-stable'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.